packages/eslint-config/lib/rules/disallow-pf-migrated-components.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @fileoverview Rule to disallow importing components already migrated to @patternfly/react-component-groups
+ * @author Filip Hlavac
+ */
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'disallow already migrated components',
+      category: 'Possible run errors',
+      recommended: true,
+    },
+    fixable: 'code',
+    schema: [],
+  },
+  create: function (context) {
+    return {
+      ImportDeclaration(node) {
+        const sourceValue = node.source.value;
+        const specifiers = node.specifiers;
+        const forbiddenImports = ['ErrorBoundary', 'NotAuthorized', 'ErrorState'];
```

Comment:
Can we move this out of the function so it's more visible and easier for us to add new deprecated values?